custom_components/ttlock/coordinator.py

```diff
@@ -38,7 +38,14 @@ class LockState:
     last_user: str | None = None
     last_reason: str | None = None
 
-    auto_lock_seconds: int = -1
+    lock_sound: bool | None = None
+
+    sensor: bool | None = None
+    opened: bool | None = None
+    sensor_battery: int | None = None
+
+    auto_lock_seconds: int | None = None
+    auto_lock: bool | None = None
```

Comment:
It might be worth computing `auto_lock` from `auto_lock_seconds` rather than maintaining both, eg

```python
@property
def auto_lock(self) -> bool | None:
  if self.auto_lock_seconds is not None:
    return self.auto_lock_seconds > 0
  else:
    return None
```

Reply:
Within the coordinator? And would that mean we would need to call the function everytime the auto_lock_seconds change? Again this is like the sensor bool that I just used to keep track of my switch, there probably isn't a need to maintain it at all 🤷